src/ia-book-actions.js

```diff
@@ -151,7 +149,6 @@ export default class IABookActions extends LitElement {
   updated(changed) {
     if (changed.has('lendingStatus') || changed.has('bwbPurchaseUrl')) {
       this.setupLendingToolbarActions();
-      this.update();
```

Comment:
moved the update request to `this.setupLendingToolbarActions`

Reply:
For learning, is this make any difference when we update request in the function where we want.?